pages/common/tee.md

```diff
@@ -12,7 +12,7 @@
 
 - Print standard input to the terminal, and also pipe it into another program for further processing:
 
-`echo "example" | tee /dev/tty | xargs printf "\033[1;34m%s\033[m\n"`
+`echo "example" | tee {{/dev/tty}} | {{xargs printf "[%s]"}}`
```

Comment:
I think `echo "example" | tee | xargs printf "[%s]"` will also do the same.
What benefit do we have from including `/dev/tty`

Reply:
That’s not correct:

<img width="439" alt="image" src="https://user-images.githubusercontent.com/282113/45855558-880f7280-bd05-11e8-9fec-97977b236407.png">

As the description says, this both prints to your terminal *and* passes it to the next command. Tee with no args does nothing.

Reply:
I don't know what was I thinking :(
You are correct, I apologise.